### PR TITLE
Migrate All content yml

### PR DESCRIPTION
**Pull request by Moejoe90**
Only thing missing is the header, do we migrate as well?

### CODE REVIEW

File: page/partials/ecosystem.ejs [hunk 1]
```diff
@@ -2,50 +2,40 @@
...
-          etc.
-        </p>
-        <div class="goto-element">Go to:</div>
+      <div class="col-md-6">
+        <p class="padme" class="content"><%= paragraph -%></p>
+        <div class="goto-element"><%= subtitle -%></div>
         <ul>
-          <li><a href="#">Neufunds Ecosystem Impact Page</a></li>
-          <li><a href="#">Neufunds Mediem Channel</a></li>
+          <li><a><%= linka1 -%></a></li>
```
Comment (krzkaczor) [inline page/partials/ecosystem.ejs]: this should also have a destination (`href`) as part of the `content.yml`.

File: page/partials/ecosystem.ejs [hunk 2]
```diff
@@ -2,50 +2,40 @@
...
+          <li><a><%= linka2 -%></a></li>
         </ul>
       </div>
 
-      <div class="col-md-7">
+      <div class="col-md-6">
```
Comment (krzkaczor) [inline page/partials/ecosystem.ejs]: why you're changing that?

Comment (Moejoe90) [inline page/partials/ecosystem.ejs]: This should be like original, your right

File: page/partials/ecosystem.ejs (hunk 1 above)
Comment (Moejoe90) [inline page/partials/ecosystem.ejs]: On that now

File: content.yml [hunk 3]
```diff
@@ -92,15 +92,21 @@ ecosystem:
     technological and legal innovation etc.
   subtitle: "Go to:"
   linka1: "Neufunds Ecosystem Impact Page"
+  linka1ref: "#"
```
Comment (krzkaczor) [inline content.yml]: please use objects and arrays! this should be like this:
```
links:
    - name: "Neufunds Ecosystem Impact Page"
    	ref: "#"
    - name: "second link"
    	ref: "#"```

Comment (krzkaczor) [inline content.yml]: this applies to other structrures as well

Comment (Moejoe90) [inline content.yml]: It can be done, but i think in the end we won't use loops to generate any of this content. 
However will do that now

**Pull request merged**